packages/salesforcedx-sobjects-faux-generator/test/integration/sobjectTransformer.test.ts

```diff
@@ -32,15 +34,15 @@ const CONNECTION_DATA = {
 };
 
 const env = createSandbox();
+const tempFolder = path.join(os.tmpdir(), PROJECT_NAME);
 
 // tslint:disable:no-unused-expression
 describe('Transform sobject definitions', () => {
   let cancellationTokenSource: CancellationTokenSource;
-  let projectPath: string;
   let emitter: EventEmitter;
 
   beforeEach(() => {
-    projectPath = path.join(process.cwd(), PROJECT_NAME);
+    env.stub(projectPaths, 'stateFolder').returns(tempFolder);
```

Comment:
instead of this test creating a random folder in the cwd I created a temp folder instead.